### PR TITLE
feat: wait 10ms to close app

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ function extendsEggApplication(egg, options) {
       return Promise.all([
         super.close(),
         this.agent.close(),
-      ]);
+      ]).then(() => new Promise(resolve => setTimeout(resolve, 10)));
```
Comment (popomore) [inline lib/app.js]: @fengmk2 是不是在 mz 里加个 wait 或 sleep

**Comment by popomore** [comment]
+ egg-mock@0.0.8

File: lib/app.js (hunk 1 above)
Comment (fengmk2) [inline lib/app.js]: 可以去提 mr，但是未必能合并。

Comment (popomore) [inline lib/app.js]: 用这个吧 https://github.com/alsotang/ko-sleep

**Pull request merged**